xbzu1/1.0/board.xml

```diff
@@ -60,37 +61,247 @@
    </jumpers>
 
    <components>   
-      <component 
-         name="part0" 
-         display_name="ZU+ MPSoC" 
-         type="fpga" 
-         part_name="xczu1cg-sbva484-1-e" 
-         pin_map_file="part0_pins.xml" 
-         vendor="xilinx" 
-         spec_url="http://avnet.me/XBoard-ZU1">
+
+      <!-- ZU+ MPSoC -->
+      <component name="part0" display_name="ZU+ MPSoC" type="fpga" part_name="xczu1cg-sbva484-1-e" pin_map_file="part0_pins.xml" vendor="xilinx" spec_url="http://avnet.me/XBoard-ZU1">
```

Comment:
This is easier to read if it is broken up on multiple lines like it was above

Reply:
Yes, I agree. Unfortunately, in my text editor, I can't collapse the line when I have it separated out onto multiple lines. For now, I have it compacted, but once we are confident the file is working, then we can go edit for readability.